open3d_slam/include/open3d_slam/Parameters.hpp

```diff
@@ -26,22 +26,19 @@ enum class IcpObjective : int {
 	PointToPlane
 };
 
-enum class MesherStrategy : int {
-	AlphaShape,
-	BallPivot,
-	Poisson
+static const std::map<std::string, IcpObjective> IcpObjectiveNames {
```

Comment:
maybe this variable can have a better name. From the given one one could think it is a vector of strings or so. I often refer to these maps as ..To...

E.g. here it could be objectiveNameToMethodMap